doc/classes/Navigation2D.xml

```diff
@@ -48,6 +52,7 @@
 			<argument index="2" name="owner" type="Object" default="null">
 			</argument>
 			<description>
+				Adds a [code]NavigationPolygon[/code] to the navigation node. Returns an ID that can be used to refer to the polygon when calling [code]navpoly_remove()[/code] or [code]navpoly_set_transform()[/code]. A [code]Transform2D[/code] that can be applied optionally applied to the polygon. Also, the polygon may be associated with an owning [code]Node[/code] (or [code]Object[/code]) for [code]get_closest_point_owner()[/code].
```

Comment:
" A [code]Transform2D[/code] that can be applied optionally applied to the polygon." - You used "applied" twice - remove the first one. Remove "that". 

Link objects and methods.